front/app/modules/commercial/id_id_card_lookup/messages.ts

```diff
@@ -37,4 +37,8 @@ export default defineMessages({
     id: 'app.modules.id_id_card_lookup.cancel',
     defaultMessage: 'Cancel',
   },
+  helpImageAltText: {
+    id: 'app.modules.id_id_card_lookup.helpImageAltText',
+    defaultMessage: 'ID card explainer',
```

Comment:
Sorry to nitpick, but it should be 'explanation' rather than 'explainer' here 😁

_explainer_ - a person who explains a concept, topic, situation, etc.

Reply:
Also, if you change messages without changing keys, it's possible that Crowdin doesn't overwrite the strings that have already been added to en.json with your new copy. You would have to check this. (I saw the old copy is still in the en.json diff).

Reply:
Hey Brent - explainer can also mean something that provides an explanation, not necessarily a person. But alright - I rarely have strong copy feeling generally so happy to change it. 🙃

Good point for the keys!